daffodil-tdml-processor/src/main/scala/org/apache/daffodil/tdml/processor/DaffodilTDMLDFDLProcessor.scala

```diff
@@ -234,21 +242,24 @@ class DaffodilTDMLDFDLProcessor private (private var dp: DataProcessor) extends
 
   override def getDiagnostics: Seq[Diagnostic] = dp.getDiagnostics
 
-  override def parse(is: java.io.InputStream, lengthLimitInBits: Long): TDMLParseResult = {
-
-    val outputter = new TDMLInfosetOutputter()
-    outputter.setBlobAttributes(blobDir, blobPrefix, blobSuffix)
+  def parse(uri: java.net.URI, lengthLimitInBits: Long): TDMLParseResult = {
+    val url = uri.toURL
+    val dpInputStream = url.openStream()
+    val saxInputStream = url.openStream()
+    doParseWithBothApis(dpInputStream, saxInputStream, lengthLimitInBits)
```

Comment:
I realize for now we're trying to test everything against both SAX and prior API. But eventually we'll just obsolete the non-SAX API, or reimplement it in terms of the SAX one, so there won't be "both". So shouldn't whether we do just SAX or both SAX and prior be under control of a constant def someplace?

Reply:
Doesn't seem unreasonable to have this configurable. We might also want to make it configurable so that when using the non-SAX API that we only generate one infoset. We currently generate like 5 infosets. For very large infosets, that could take up qutie a bit of memory.

Maybe we create a new bug to make this all configuratble some how and have some low-memory/overhead option?

Reply:
Created [DAFFODIL-2392](https://issues.apache.org/jira/browse/DAFFODIL-2392) for this